### ISSUE TITLE
Sorting by variance in consensus view is not sorting

### ISSUE THREAD

**Issue by seanperkins** [MobilityLabs/pdredesign-server#257]
Reported by Bryan so would be good to look at his giant assessments. It may just be that there is no loading indicator.

**Comment by MakotoTheKnight** [comment]
Looks like it's a bit more complex than this; we do have the numbers coming back correctly but for whatever reason the client doesn't seem to be displaying them.  While we used to do this sort of sorting in the client _anyway_, I wonder if this would be a good opportunity to push this logic down into the server side to reduce the burden on the client.

**Comment by MakotoTheKnight** [comment]
Okay, I've figured this one out.  The variance sort doesn't work because there have been changes to the UI to cause it to not be supported.  We list all of our content explicitly by category, which doesn't imply that the questions are in any variance-sorted order.

We can change this so that when we reorder the data, we also set a condition that causes the parent directive to render in variance-order.  We'll need to take _great_ care here so we don't draw the DOM multiple times per request, especially with large data sets.
